### PR TITLE
Remove quickstart-subnet location patch

### PR DESCRIPTION
**Pull request by jaymiracola**
This is an unnecessary patch that breaks the composition.

### CODE REVIEW

File: content/master/getting-started/provider-azure-part-2.md [hunk 1]
```diff
@@ -401,15 +401,6 @@ spec:
                   matchControllerRef: true
                 resourceGroupNameSelector:
                   matchControllerRef: true
-          patches:
```
Comment (jbw976) [inline content/master/getting-started/provider-azure-part-2.md:404]: ah interesting, it does look like `Subnet` is the only resource in this composition that doesn't have a `spec.forProvider.location` field. Nice catch!

https://marketplace.upbound.io/providers/upbound/provider-azure-network/v1.5.0/resources/network.azure.upbound.io/Subnet/v1beta1

**Review by jbw976 — COMMENTED** [review]
Change looks good from cross referencing the resource with its entry in the marketplace. Go ahead and copy this content to the `content/v1.17` directory also so it shows up in the `latest` docs. Thanks @jaymiracola! 🙇‍♂️

**Comment by jbw976** [comment]
Oh and also please add the DOC sign-off to your commit too!

https://github.com/crossplane/crossplane/tree/main/contributing#certificate-of-origin

**Comment by jbw976** [comment]
yep exactly @jaymiracola, from https://github.com/crossplane/docs/pull/812/checks:

> Commit sha: [8fa447c](https://github.com/crossplane/docs/pull/812/commits/8fa447c4bf64b1a8e8b5eaee0a86e92a75b275d0), Author: Jay Miracola, Committer: GitHub; The sign-off is missing.

You could amend that first commit also, or just squash them into a single commit with a sign-off. Let me know if you'd like any help with that!

**Comment by jaymiracola** [comment]
Thanks @jbw976 ! hopefully its all set now 😅

**Review by jbw976 — APPROVED** [review]

**Pull request merged**